src/pyOpenMS/pxds/AcquisitionInfo.pxd

```diff
@@ -19,8 +26,9 @@ cdef extern from "<OpenMS/METADATA/AcquisitionInfo.h>" namespace "OpenMS":
         String getMethodOfCombination() nogil except +
         void setMethodOfCombination(String method) nogil except +
 
-        Size size() nogil except + #wrap-doc:Number a Acquisition objects
+        Size size() nogil except +
```

Comment:
why has doc string been removed?

Reply:
mh, must have been some funny business going on whilemerging. Def not on purpose, see my comment before.